<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>577</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>977</width>
    <height>577</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>977</width>
    <height>577</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Launcher V1 Atidote</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>490</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>495</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>File downloaded: </string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>500</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>000000000000000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>490</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>100 / 100</string>
    </property>
   </widget>
   <widget class="QWebView" name="webView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>977</width>
      <height>561</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>http://88.179.239.14:8080/2.0/</string>
     </url>
    </property>
    <property name="zoomFactor">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <zorder>webView</zorder>
   <zorder>pushButton</zorder>
   <zorder>progressBar</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionRepair_Game"/>
    <addaction name="actionRemove_installation"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuOption"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRepair_Game">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Images/Image/kservices-icone-9072-32.png</normaloff>:/Images/Image/kservices-icone-9072-32.png</iconset>
   </property>
   <property name="text">
    <string>Repair Game</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Images/Image/fermer-croix-supprimer-erreurs-sortie-icone-4368-32.png</normaloff>:/Images/Image/fermer-croix-supprimer-erreurs-sortie-icone-4368-32.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRemove_installation">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Images/Image/supprimer-vide-ordures-corbeille-icone-7418-32.png</normaloff>:/Images/Image/supprimer-vide-ordures-corbeille-icone-7418-32.png</iconset>
   </property>
   <property name="text">
    <string>Remove installation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myres.qrc"/>
 </resources>
 <connections/>
</ui>
